packages/components/package.json

```diff
@@ -44,6 +44,7 @@
 		"re-resizable": "^4.7.1",
 		"react-click-outside": "^3.0.0",
 		"react-dates": "^17.1.1",
+		"react-spring": "^8.0.20",
```

Comment:
Is `react-spring` something we might consider adding as a library in WP core?  I imagine a number of plugin/theme author's might use it as well (I'm currently using it for a block I'm developing).

Reply:
I personally worry about adding this kind of libs as WP core libs because of the breaking changes they might introduce. For the moment I think it's small enough to "allow it" to be duplicated but I think that's a good question and we should probably have some defined guidelines to answer it.